browse/controllers/abs_page/__init__.py

```diff
@@ -120,11 +128,9 @@ def get_abs_page(arxiv_id: str,
     except IdentifierException:
         raise AbsNotFound(data={'arxiv_id': arxiv_id})
     except (AbsException, Exception) as e:
-        print(f'Problem: {e}')
```

Comment:
It would probably be good to log this too.

Reply:
This gets changed in another branch to go through to the arxiv-base 500 handler.